app/src/main-process/squirrel-updater.ts

```diff
@@ -92,12 +98,20 @@ function writeBatchScriptCLITrampoline(binPath: string): Promise<void> {
   return writeFile(trampolinePath, trampoline)
 }
 
+/** We have to make sure the versioned path in the script has the correct format
+ * even if its Windows otherwise there can be edge cases with WSL see issue:
+ * https://github.com/desktop/desktop/issues/4998
+ */
```

Comment:
Love this comment but I think it would make more sense to have it be a regular code comment and to put it right next to the `.replace` call. We can skip the bit about "even if its Windows" (see below) and it'd be great if we could clarify what the "correct format" is. Perhaps something like

```
// The path we get from `resolveVersionedPath` is a Win32 relative
// path (something like `..\app-2.5.0\resources\app\static\github.sh`).
// We need to make sure it's a POSIX path in order for WSL to be able
// to resolve it. See https://github.com/desktop/desktop/issues/4998